front/app/containers/UsersShowPage/Surveys/SurveySubmissionCard/messages.ts

```diff
@@ -0,0 +1,8 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  downloadYourResponses: {
+    id: 'app.containers.UsersShowPage.Surveys.SurveySubmissionCard.downloadYourResponses',
+    defaultMessage: 'Download your responses',
```

Comment:
I think we should perhaps give an indication of the format here. `Download your responses (.xlsx)`?